pkg/console/install_panels.go

```diff
@@ -41,7 +41,7 @@ const (
 var (
 	once          sync.Once
 	userInputData = UserInputData{
-		NTPServers: "ntp.ubuntu.com",
+		NTPServers: "0.suse.pool.ntp.org",
```

Comment:
is there any reference of why we choose the `0.suse.pool.ntp.org`, thanks, it seems like subdomain to me

Reply:
Yes, I saw this domain when I install openSUSE. Also, it is default `FallbackNTPServers` in harvester-v0.3.0-preview1.

![Screen Shot 2021-09-23 at 12 47 31 PM](https://user-images.githubusercontent.com/4927361/134455943-f9a2b5c2-ef45-42ad-9127-dca22ef4cc6f.png)


![Screen Shot 2021-09-23 at 12 37 44 PM](https://user-images.githubusercontent.com/4927361/134455248-1e0ade49-d19c-40be-bc09-7c673cbc8a40.png)

Reply:
I think we should use the pool ntp server config instead of a single server node, can we check it this works.
https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-ntp.html#sec-netz-xntp-netconf

Reply:
Do you mean use `pool.ntp.org` as default input? I have checked `suse.pool.ntp.org`. We don't have this domain. We can also use `0.suse.pool.ntp.org,1.suse.pool.ntp.org,2.suse.pool.ntp.org` as default input.

Reply:
Yep, does the `pool.ntp.org` works? if not probably default to the above multi ntp server config like 0,1,2 is much better, thanks

Reply:
I think 0.suse.pool.ntp.org is fine. It resolves to a set of timeservers

Reply:
@guangbochen yes, `pool.ntp.org` works. Also, both `pool.ntp.org` and `0.suse.pool.ntp.org` can be resolved to a set of timeservers. Do we need to change default input to `pool.ntp.org`? Thank you.

Reply:
if it will be resolved to many timeservers then let's just use the SUSE one for now, thanks